directives/annotated.py

```diff
@@ -153,10 +175,14 @@ def run(self):
         env.annotated_annotation_count += 1
         node['annotation-number'] = env.annotated_annotation_count
         node['name-of-annotated-section'] = env.annotated_name
+        if self.arguments:
+            node['replacement'] = self.arguments[0]
         return [node]
 
 def visit_annotation_node(self, node):
-    self.body.append('<div class="container codecomment comment-%s-%s">' % (node['name-of-annotated-section'], node['annotation-number']))
+    replacement_attrib = ' data-replacement="' + node['replacement'] + '"' if 'replacement' in node else ""
+    html_bits = (node['name-of-annotated-section'], node['annotation-number'], replacement_attrib)
+    self.body.append('<div class="container codecomment comment-%s-%s"%s>' % html_bits)
```

Comment:
The same thing about HTML escaping as above: https://docs.python.org/3/library/html.html#html.escape

Reply:
Hmm, unlike the above (which is an inlined annotation), this is a bona-fide RST directive, so its argument doesn't get parsed. So, yeah, escaping could be an issue if someone wants to use double-quotes here.

Reply:
Indeed. The annotation argument must be HTML-escaped. Double quotes absolutely break the HTML attribute. Other special HTML characters must be escaped too since there are cases in which they mess up HTML parsers inside attributes. Calling `html.escape()` is enough.